malasakit-django/pcari/views.py

```diff
@@ -110,8 +107,14 @@ def make_quantitative_question_ratings(respondent, responses):
 def make_comments(respondent, responses):
     for question_id, message in responses.get('comments', {}).iteritems():
         question = QualitativeQuestion(id=int(question_id))
-        yield Comment(respondent=respondent, question=question,
-                      language=respondent.language, message=message)
+
+        # Replaces empty messages with None so they can show up as placeholders in admin 
+        if message == "" or message.isspace():
```

Comment:
When preprocessing a message, use the [`strip` function](https://docs.python.org/2/library/string.html#string.strip) to clean up the string. (`'...'.strip()` removes all contiguous leading and trailing whitespace, so `' \t alpha \t '.strip()` is `'alpha'`.)